projects/shared/src/lib/materia.json

```diff
@@ -1,338 +1,791 @@
-    },
-    {
-        "name": "Mystify",
-        "ty": "magic"
-    },
-    {
-        "name": "Time",
-        "ty": "magic"
-    },
-    {
-        "name": "Gravity",
-        "ty": "magic"
-    },
-    {
-        "name": "Destruct",
-        "ty": "magic"
-    },
-    {
-        "name": "Barrier",
-        "ty": "magic"
-    },
-    {
-        "name": "Exit",
-        "ty": "magic"
-    },
-    {
-        "name": "Comet",
-        "ty": "magic"
-    },
-    {
-        "name": "Full Cure",
-        "ty": "magic"
-    },
-    {
-        "name": "Contain",
-        "ty": "magic"
-    },
-    {
-        "name": "Ultima",
-        "ty": "magic"
-    },
-    {
-        "name": "Shield",
-        "ty": "magic"
-    },
-    {
-        "name": "Master Magic",
-        "ty": "magic"
-    },
-    {
-        "name": "Choco/Mog",
-        "ty": "summon"
-    },
-    {
-        "name": "Shiva",
-        "ty": "summon"
-    },
-    {
-        "name": "Ifrit",
-        "ty": "summon"
-    },
-    {
-        "name": "Ramuh",
-        "ty": "summon"
-    },
-    {
-        "name": "Titan",
-        "ty": "summon"
-    },
-    {
-        "name": "Odin",
-        "ty": "summon"
-    },
-    {
-        "name": "Kajata",
-        "ty": "summon"
-    },
-    {
-        "name": "Titan",
-        "ty": "summon"
-    },
-    {
-        "name": "Odin",
-        "ty": "summon"
-    },
-    {
-        "name": "Kjata",
-        "ty": "summon"
-    },
-    {
-        "name": "Bahamut",
-        "ty": "summon"
-    },
-    {
-        "name": "Alexander",
-        "ty": "summon"
-    },
-    {
-        "name": "Neo Bahamut",
-        "ty": "summon"
-    },
-    {
-        "name": "Leviathan",
-        "ty": "summon"
-    },
-    {
-        "name": "Phoenix",
-        "ty": "summon"
-    },
-    {
-        "name": "Hades",
-        "ty": "summon"
-    },
-    {
-        "name": "Bahamut ZERO",
-        "ty": "summon"
-    },
-    {
-        "name": "Typoon",
-        "ty": "summon"
-    },
-    {
-        "name": "Knights of the Round",
-        "ty": "summon"
-    },
-    {
-        "name": "All",
-        "ty": "support"
-    },
-    {
-        "name": "Elemental",
-        "ty": "support"
-    },
-    {
-        "name": "Added Effect",
-        "ty": "support"
-    },
-    {
-        "name": "MP Absorb",
-        "ty": "support"
-    },
-    {
-        "name": "HP Absorb",
-        "ty": "support"
-    },
-    {
-        "name": "Added Cut",
-        "ty": "support"
-    },
-    {
-        "name": "MP Turbo",
-        "ty": "support"
-    },
-    {
-        "name": "Steal As Well",
-        "ty": "support"
-    },
-    {
-        "name": "Magic Counter",
-        "ty": "support"
-    },
-    {
-        "name": "Final Attack",
-        "ty": "support"
-    },
-    {
-        "name": "Quadra Magic",
-        "ty": "support"
-    },
-    {
-        "name": "Sneak Attack",
-        "ty": "support"
+        "clr": "Green",
+        "ty": "Magic",
+        "eff": [
```

Comment:
I'd like to keep fields as full words when possible.  "type" is "ty" as it conflicts with keywords in many languages.

Reply:
Ah, I thought you were prioritizing brevity and purposefully had used shorter names. That's reverted back to full name snow.